tempesta_fw/http_parser.c

```diff
@@ -1237,6 +1207,63 @@ __req_parse_x_forwarded_for(TfwHttpMsg *msg, unsigned char *data, size_t len)
 	return r;
 }
 
+/* In request we should pass empty headers:
+ * RFC 7230 5.4:
+ * ....
+ * ....
+ * If the authority component is missing or
+ * undefined for the target URI, then a client MUST send a Host header
+ * field with an empty field-value.
+ *
+ * ATTENTION: using of RGEN_LWS_empty should be corresponding with
+ * tfw_http_msg_hdr_val function
```

Comment:
Please use `NOTE:`. Also, the test is not clear. Did you mean something like the following?
```The use of RGEN_LWS_empty() should be matched with the BUG_ON() statements in tfw_http_msg_hdr_val() function```.